third_party/Makefile

```diff
@@ -239,3 +239,16 @@ ifneq (,$(wildcard msccl/tests/msccl-tests-nccl/Makefile))
 	mkdir -p $(SB_MICRO_PATH)/bin/msccl-tests-nccl && \
 	cp -r -v ./msccl/tests/msccl-tests-nccl/build/* $(SB_MICRO_PATH)/bin/msccl-tests-nccl/
 endif
+
+# Build nvbandwidth.
+REQUIRED_CMAKE_VERSION := 3.20.0
+CMAKE_VERSION := $(shell cmake --version 2>/dev/null | head -n 1 | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+nvbandwidth: sb_micro_path
+	@if [ "$(CMAKE_VERSION)" = "" ]; then \
+        echo "Skip nvbandwidth for CMake is not installed."; \
```

Comment:
I think we should expose thrid-party dependency issues in this file instead of skipping it. Maybe we don't need to check cmake version here.